### DISCUSSION TITLE
Is College ID card accepted

### DISCUSSION THREAD

**Discussion by HallowSiddharth** [community/community#134776] [GitHub Education]
### Hi everyone,

Is College ID card accepted?

**Comment by v1sh-afk** [comment]
Yeah, I think you can give your college ID card as an ID proof, not sure exactly though. Do lmk if theres any update

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/134781#discussioncomment-10246657) for why this discussions was closed and locked.
